app/controllers/projects/samples/metadata_controller.rb

```diff
@@ -9,7 +9,8 @@ def update # rubocop:disable Metrics/AbcSize
         respond_to do |format|
           metadata_fields = ::Samples::Metadata::UpdateService.new(@project, @sample, current_user,
                                                                    metadata_params).execute
-          if !metadata_fields.nil? && metadata_fields[:updated].count.positive?
+          changed_metadata_fields = metadata_fields[:added] + metadata_fields[:updated] + metadata_fields[:deleted]
```

Comment:
Can we rename this variable to `modified_metadata`?

Reply:
Fixed in 97f16f77456fa6b0b755930801a23fc43a3670d6